packages/ketcher-macromolecules/src/state/rna-builder/rnaBuilderSlice.ts

```diff
@@ -308,7 +335,12 @@ export const selectActivePresetMonomerGroup = (state: RootState) =>
 
 export const selectIsPresetReadyToSave = (preset: IRnaPreset): boolean => {
   return Boolean(
-    (preset.phosphate || preset.sugar || preset.base) && preset.name,
+    (preset.phosphate || preset.sugar || preset.base) &&
+      preset.name &&
+      Number(!!preset.phosphate) +
+        Number(!!preset.sugar) +
+        Number(!!preset.base) >=
+        2,
```

Comment:
Let's discuss this check. Need to check it from business perspective.

Reply:
Updated validations, after adding new validation (check preset structure) - part about checking how many items in preset is not needed anymore, therefore removed it